### ISSUE TITLE
appears to be a crash in PFT code

### ISSUE THREAD

**Issue by schweitzpgi** [flang-compiler/f18-llvm-project#74]
Taking lapack from http://www.netlib.org/lapack/lapack-lite-3.1.0.tgz

Compile command of `cd SRC; bbc stgevc.f`
Also see `bbc slaebz.f`, `dlaebz.f`, etc.

We get a stack trace pointing to PFT.

```
 #9 0x0000000002148d09 Fortran::lower::pft::Evaluation::isIntermediateConstructStmt() const /f18-llvm-project/flang/include/flang/Lower/PFTBuilder.h:222:5
#10 0x00000000020e9c33 (anonymous namespace)::FirConverter::genFIR(Fortran::lower::pft::Evaluation&, bool) /f18-llvm-project/flang/lib/Lower/Bridge.cpp:1598:16
```

**Comment by schweitzpgi** [comment]
I'm seeing this failure on an ubuntu machine when compiling with clang 9.0 and a centos machine with g++ 8.3.

**Comment by schweitzpgi** [comment]
Merged in the workaround #79 for now.
